command_line_rock_paper_scissors.py

```diff
@@ -1,9 +1,10 @@
 """Object-Oriented Rock Paper Scissors"""
 import random
+import sys
 
 from rock_paper_scissors.user import User
-
 from rock_paper_scissors.game import Game
+from rock_paper_scissors import exc
```

Comment:
This one should come before `rock_paper_scissors.game`.

A quick clarification on the alphabetical ordering of imports, since I don't think it's always immediately obvious: within each block, the ordering applies first to packages, then subpackages and module names, and finally the actual value(s) being imported.

Example:
```python
import abc
from argparse import (
    ArgumentParser,
    HelpFormatter
)
from multiprocessing.connection import (
    Connection,
    Socket
)
from multiprocessing import Event
from multiprocessing.util import info
from os import path